pkg/psmdb/getters.go

```diff
@@ -67,7 +68,24 @@ func GetRSPods(ctx context.Context, k8sclient client.Client, cr *api.PerconaServ
 			return rsPods, errors.Wrap(err, "failed to list pods")
 		}
 
-		rsPods.Items = append(rsPods.Items, compPods.Items...)
+		rs := cr.Spec.Replset(rsName)
+		var podsLen int
+		switch sts.Labels["app.kubernetes.io/component"] {
+		case "arbiter":
+			podsLen = int(rs.Arbiter.Size)
```

Comment:
Since we are looping through all stateful sets related to `rsName`, why do you need `podsLen`? To get the pod count, can't you simply use like `sts.Size()`?

Reply:
I think you are right https://github.com/percona/percona-server-mongodb-operator/pull/1323/commits/8692f487bbabe5db594a0fed1a5b56382f14980d